source/configuration/read.txt

```diff
@@ -330,9 +330,13 @@ Change Streams
      - | Specifies whether to publish the changed document or the full
          change stream document.
        |
-       | When set to ``true``, the connector filters out messages that
+       | When set to ``true``:
+       
+       - The connector filters out messages that
          omit the ``fullDocument`` field and only publishes the value of the
          field.
+       - If you don't specify a schema, the connector infers the schema
+         from the change stream document rather than from the underlying collection.
```

Comment:
I: per the [style guide re: lists](https://www.mongodb.com/docs/meta/style-guide/style/lists/), do not start a list with a fragment

```suggestion
       | The connector displays the following behavior when this property is set to ``true``:
       
       - The connector filters out messages that
         omit the ``fullDocument`` field and only publishes the value of the
         field.
       - If you don't specify a schema, the connector infers the schema
         from the change stream document rather than from the underlying collection.
```

Reply:
great catch